charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```diff
@@ -0,0 +1,108 @@
+{{- if .Values.controller.enabled }}
```

Comment:
I don't think there is ever a case when the controller can be disabled, the server, repo-server and controller they all work together. So I don't see any point in configuring enable/disable of it.

Reply:
Some charts, such as the prometheus operator allow you to enable or disable central parts of the system.  The use-case I can imagine is deploying different parts of ArgoCD in different sub-carts.  I don't think that is a very strong argument and so for simplicity  sake I've removed these items.
controller enabled: https://github.com/argoproj/argo-helm/pull/129/commits/4fe65a4e0932e66155e21c485ad04df41098249a
repo server enabled: https://github.com/argoproj/argo-helm/pull/129/commits/41df9ed6725dd7ece14944bc8daff335dd365b15
server enabled: https://github.com/argoproj/argo-helm/pull/129/commits/ca53ad463a24ee62e496e50cd5346744ac31fdd8